package.json

```diff
@@ -21,7 +21,9 @@
         "@mojaloop/central-services-logger": "^10.6.0",
```

Comment:
there is a need to fix `name`, `author`, `repository` properties!

Reply:
done